spring-xd-test-fixtures/src/main/java/org/springframework/xd/test/fixtures/KafkaSink.java

```diff
@@ -54,18 +54,18 @@ public static KafkaSink withDefaults() {
 
 	@Override
 	protected String toDSL() {
-		return String.format("kafka --brokerList=%s --topic=%s", brokerList, topic);
+		return String.format("kafka --brokerList=%s --topic=%s", zkConnect, topic);
 	}
 
 	/**
 	 * Set the brokerList for the sink
 	 *
-	 * @param brokerList A list of brokers that the sink can connect
+	 * @param zkConnect A list of brokers that the sink can connect
 	 * @return instance of Kafka sink
 	 */
-	public KafkaSink brokerList(String brokerList) {
-		Assert.hasText(brokerList, "brokerList must not be empty nor null");
-		this.brokerList = brokerList;
+	public KafkaSink brokerList(String zkConnect) {
+		Assert.hasText(zkConnect, "brokerList must not be empty nor null");
```

Comment:
I forgot to update 'brokerList' to zkConnect  ... should be consistent everywhere....